airflow/gcp/operators/gcs.py

```diff
@@ -625,3 +625,36 @@ def execute(self, context: Dict):
                 object_name=self.destination_object,
                 filename=destination_file.name
             )
+
+
+class GCSDeleteBucketOperator(BaseOperator):
+    """
+    Deletes bucket from a Google Cloud Storage, either
+
+    :param bucket_name: The GCS bucket to delete from
```

Comment:
```suggestion
    :param bucket_name: The name of the bucket you want to delete.
```

Reply:
I changed to: `:param bucket_name: name of the bucket which will be deleted'
I think this version will be more clear and simple